src/wp-includes/class-phpass.php

```diff
@@ -173,8 +173,8 @@ function gensalt_blowfish($input)
 		$itoa64 = './ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz0123456789';
 
 		$output = '$2a$';
-		$output .= chr(ord('0') + $this->iteration_count_log2 / 10);
-		$output .= chr(ord('0') + $this->iteration_count_log2 % 10);
+		$output .= chr((int)(ord('0') + $this->iteration_count_log2 / 10));
+		$output .= chr((int)(ord('0') + $this->iteration_count_log2 % 10));
```

Comment:
The `(int)` cast shouldn't be needed for this last line as the result from using the modulo operator will always be an integer.

As per the [PHP docs](https://www.php.net/manual/en/language.operators.arithmetic.php):
> Operands of modulo are converted to int before processing. For floating-point modulo, see [fmod()](https://www.php.net/manual/en/function.fmod.php).

Reply:
Removed in [f95e983](https://github.com/WordPress/wordpress-develop/pull/4049/commits/f95e9830cda193963c4d5e3ce3c953e191497409).